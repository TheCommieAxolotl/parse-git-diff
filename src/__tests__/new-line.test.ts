import { getFixture } from './test-utils';
import parseGitDiff from '../parse-git-diff';

describe('new-line', () => {
  const fixture = getFixture('new-line');

  it('parse `new-line`', () => {
    expect(parseGitDiff(fixture)).toMatchSnapshot();
  });
});
