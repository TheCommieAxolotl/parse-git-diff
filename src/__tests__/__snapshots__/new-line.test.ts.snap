// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`new-line parse \`new-line\` 1`] = `
{
  "files": [
    {
      "chunks": [
        {
          "changes": [
            {
              "content": "newfile",
              "lineAfter": 1,
              "lineBefore": 1,
              "type": "UnchangedLine",
            },
            {
              "content": "newline",
              "lineAfter": 2,
              "type": "AddedLine",
            },
          ],
          "fromFileRange": {
            "lines": 1,
            "start": 1,
          },
          "toFileRange": {
            "lines": 2,
            "start": 1,
          },
          "type": "Chunk",
        },
      ],
      "path": "rename.md",
      "type": "ChangedFile",
    },
  ],
  "type": "GitDiff",
}
`;
